### PR TITLE
Custom.css does not exist

### PR DESCRIPTION
**Pull request by nickumia-reisys**
Related to
- https://github.com/GSA/data.gov/issues/4262

Since CKAN 2.9, web assets are bundled in the `fanstatic/webassets.yml` and compiled to a single file.  The reference to the file is the yaml tag given in the `webassets.yml`.  `custom.css` was bundled into `styles` which is included in the `templates/base.html` and no other references are needed since everything `ckan_extends` everything else.

### CODE REVIEW

**Review by btylerburton — DISMISSED** [review]
📉

**Review by btylerburton — DISMISSED** [review]
Safer and less work.

**Comment by nickumia-reisys** [comment]
I'm convinced these tests failing have nothing to do with the code change...

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ARG CKAN_VERSION=2.9
-FROM openknowledge/ckan-dev:${CKAN_VERSION}
+FROM openknowledge/ckan-dev:${CKAN_VERSION}.7
```
Comment (btylerburton) [inline Dockerfile:2]: is this supposed to be there?

Comment (nickumia-reisys) [reply Dockerfile:2]: That is the temporary fix.  This extension does not work with anything newer right now...

**Review by jbrown-xentity — APPROVED** [review]

**Review by btylerburton — APPROVED** [review]

**Pull request merged**